docs/resources/monitor.md

```diff
@@ -41,176 +44,73 @@ resource "datadog_monitor" "foo" {
--   `priority` (Optional) Integer from 1 (high) to 5 (low) indicating alert severity.
--   `silenced` (Optional) Each scope will be muted until the given POSIX timestamp or forever if the value is 0. Use `-1` if you want to unmute the scope. **Deprecated** The `silenced` parameter is being deprecated in favor of the downtime resource. This will be removed in the next major version of the Terraform Provider.
-
-    To mute the alert completely:
-
-          silenced = {
-            "*" =  0
-          }
-
-    To mute role:db for a short time:
-
-          silenced = {
-            "role:db" = 1412798116
-          }
-
-    Note: due to [HCL limitations](https://github.com/hashicorp/terraform/issues/2042), it is impossible to use interpolations in keys. For example, the following will result in muting the scope `role:${var:role}` (no interpolation is done):
-
-          silenced = {
-              "role:${var:role}" = 0
-          }
-
-    To workaround this, you can use the [map function](https://www.terraform.io/docs/configuration/functions/map.html) of HCL:
-
-          silenced = ${map("role:${var:role}", 0)}
-
-## Silencing by Hand and by Downtimes
-
-There are two ways how to silence a single monitor:
-
--   Mute it by hand
--   Create a Downtime
-
-Both of these actions add a new value to the `silenced` map. This can be problematic if the `silenced` attribute doesn't contain them in your Terraform, as they would be removed on next `terraform apply` invocation. In order to prevent that from happening, you can add following to your monitor:
-
-```hcl
-lifecycle {
-  ignore_changes = [silenced]
-}
-```
-
-The above will make sure that any changes to the `silenced` attribute are ignored.
-
-This issue doesn't apply to multi-monitor downtimes (those that don't contain `monitor_id` ), as these don't influence contents of the `silenced` attribute.
-
-## Attributes Reference
-
-The following attributes are exported:
-
--   `id`: ID of the Datadog monitor
+- **critical** (String)
+- **critical_recovery** (String)
+- **ok** (String)
+- **unknown** (String)
+- **warning** (String)
+- **warning_recovery** (String)
 
 ## Import
 
-Monitors can be imported using their numeric ID, e.g.
+Import is supported using the following syntax:
 
-```console
-$ terraform import datadog_monitor.bytes_received_localhost 2081
-```
-
-## Composite Monitors
-
-You can compose monitors of all types in order to define more specific alert conditions (see the [doc](https://docs.datadoghq.com/monitors/monitor_types/composite/)). You just need to reuse the ID of your `datadog_monitor` resources. You can also compose any monitor with a `datadog_synthetics_test` by passing the computed `monitor_id` attribute in the query.
-
-```hcl
-resource "datadog_monitor" "bar" {
-  name = "Composite Monitor"
-  type = "composite"
-  message = "This is a message"
-
-  query = "${datadog_monitor.foo.id} || ${datadog_synthetics_test.foo.monitor_id}"
-}
```

Comment:
Probably worth keeping this and adding into the hardcoded monitors guide page instead